pkgs/development/python-modules/natasha/default.nix

```
@@ -0,0 +1,32 @@
+{ buildPythonPackage, fetchPypi, python, pymorphy2, razdel, navec, slovnet
+, yagry, ipymarkup, pytest, isPy3k, lib }:
```

```suggestion
{ lib
, buildPythonPackage
, ... 
}:
```
Would you please add a line per item for future diff?

---

Done!